frontend/src/lib/components/chart/chart-page/encoding/chart-encoding/encoding-components/MetricsEncodingMultiChoice.svelte

```diff
@@ -1,55 +1,53 @@
 <script lang="ts">
 	import { metrics } from '$lib/stores';
-	import { svelecteRenderer } from '$lib/util/util';
 	import Checkbox from '@smui/checkbox';
-	import Svelecte from 'svelecte';
 	import { createEventDispatcher } from 'svelte';
-	import { dndzone } from 'svelte-dnd-action';
-	import EncodingContainer from './EncodingContainer.svelte';
+	import { MultiSelect } from 'svelte-multiselect';
 
 	export let numberValues: number[];
 
 	const dispatch = createEventDispatcher<{ selected: number[] }>();
 
 	let options: { value: number; label: string }[] = [];
-	let value: number[] = [];
+	let value: { value: number; label: string }[] = [];
```

Comment:
Should value in this object really be named id? It normally corresponds to id, right? would make it easier to understand.